src/views/GraphDetail.vue

```diff
@@ -93,7 +93,7 @@
                 v-for="app in apps"
                 :key="app.name"
                 class="pl-2"
-                :href="`${app.url}/?workspace=${workspace}&graph=${graph}`"
+                :href="`${app.url}/?workspace=${workspace}&graph=${graph}&token=${userToken}`"
```

Comment:
I'd prefer if this was `loginToken`, so it's consistent with the other use of it.

```suggestion
                :href="`${app.url}/?workspace=${workspace}&graph=${graph}&loginToken=${userToken}`"
```

Reply:
Also, it might be good to have a computed function for the end `&loginToken=...`, which would return the empty string if the token isn't present. This would prevent always passing `&loginToken=` whenever the user isn't logged in.

Reply:
See 89f51e5